src/source/CppGenerator.scala

```diff
@@ -255,11 +255,11 @@ class CppGenerator(spec: Spec) extends Generator(spec) {
   override def generateInterface(origin: String, ident: Ident, doc: Doc, typeParams: Seq[TypeParam], i: Interface) {
     val refs = new CppRefs(ident.name)
     i.methods.map(m => {
-      m.params.map(p => refs.find(p.ty))
-      m.ret.foreach(refs.find)
+      m.params.map(p => refs.find(p.ty, true))
+      m.ret.foreach((x)=>refs.find(x,true))
```

Comment:
Super-nitpick, but you need a space after the comma.  It only jumps out at me because it's different on two adjacent lines. :)